pkgs/development/python-modules/huggingface-hub/default.nix

```
@@ -13,7 +13,7 @@
 
 buildPythonPackage rec {
   pname = "huggingface-hub";
-  version = "0.20.3";
+  version = "0.21.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
Let's replace the legacy parameter.

---

Good idea, done !